src/BloomExe/Edit/EditingView.cs

```diff
@@ -244,9 +244,13 @@ private void ChangeBookMetadata(Metadata metadata)
 			//note that the only way currently to recognize a custom license is that RightsStatement is non-empty while description is empty
 			var rights = metadata.License.RightsStatement;
 			metadata.License.RightsStatement = rights;
-			string idOfLanguageUsed;
 
+			string idOfLanguageUsed;
 			string description = metadata.License.GetDescription(_model.LicenseDescriptionLanguagePriorities, out idOfLanguageUsed).Replace("'", "\\'");
+			// BL-1179: For a Custom license, GetDescription just returns the RightsStatement in language "und"; don't duplicate it.
```

Comment:
Is there another way to detect a Custom license?  I'm pragmatic enough to let this pass, but would prefer some better way to detect the need for clearing/ignoring the description.